### ISSUE TITLE
Xiaomi Aqara Plug EU lumi.plug.maeu01 shows wrong power values

### ISSUE THREAD

**Issue by grimmaldus** [dresden-elektronik/deconz-rest-plugin#6372]
## Describe the bug

Power sensors of Aqara power plugs EU show readings that are 10x higher than actual power consumption.

## Steps to reproduce the behavior

Use version deCONZ  2.18.2 with ConBee II fwversion 266B0700.

## Expected behavior

Expected value 10x lower than actual.

## Screenshots

![deconz_aquara_plug](https://user-images.githubusercontent.com/9221276/193377597-6d3c9144-c5e9-4159-b72b-5a88dbd23347.png)
## Environment

 - Host system: NAS
 - Running method:  Home Assistent deCONZ Add-on
 - Firmware version: (266B0700)
 - deCONZ version: (2.18.2)
 - Device: ConBee II

**Comment by Smanar** [comment]
There is a DDF for this device but only enabled for some versions.


>   "comment": "DDF for device firmwares at least above 0.0.0_0022, paired with xBee I fw 0x26400500 / xBee II fw 0x266f0700 and above",
>   "matchexpr": "R.endpoints.includes(0x15) && R.endpoints.includes(0x1F)",

Do you have endpoint 0x15 and 0x1F on your ? it's visible on the GUI.
Else try direclty to edit the DDF devices\xiaomi\xiaomi_sp-euc01_smart_plug.json

```
        {
          "name": "state/power",
          "refresh.interval": 10,
          "read": {
            "at": "0x0055",
            "cl": "0x000C",
            "ep": 21,
            "fn": "zcl"
          },
          "parse": {
            "at": "0x0055",
            "cl": "0x000C",
            "ep": 21,
            "eval": "Item.val = Math.round(Attr.val);"
          }
        },
```

replace the "eval" line by "eval": "Item.val = Math.round(Attr.val / 10);"

**Comment by Smanar** [comment]
Perhaps some infos here https://forum.phoscon.de/t/aqara-powersocket-consumption-issues/2494/5

**Comment by grimmaldus** [comment]
> There is a DDF for this device but only enabled for some versions.
> 
> > "comment": "DDF for device firmwares at least above 0.0.0_0022, paired with xBee I fw 0x26400500 / xBee II fw 0x266f0700 and above",
> > "matchexpr": "R.endpoints.includes(0x15) && R.endpoints.includes(0x1F)",
> 
> Do you have endpoint 0x15 and 0x1F on your ? it's visible on the GUI. Else try direclty to edit the DDF devices\xiaomi\xiaomi_sp-euc01_smart_plug.json
> 
> ```
>         {
>           "name": "state/power",
>           "refresh.interval": 10,
>           "read": {
>             "at": "0x0055",
>             "cl": "0x000C",
>             "ep": 21,
>             "fn": "zcl"
>           },
>           "parse": {
>             "at": "0x0055",
>             "cl": "0x000C",
>             "ep": 21,
>             "eval": "Item.val = Math.round(Attr.val);"
>           }
>         },
> ```
> 
> replace the "eval" line by "eval": "Item.val = Math.round(Attr.val / 10);"

Thank you, I've already fixed it in similar fashion. I just wanted to report it as a bug since it has introduced in 2.18.2.

**Comment by Smanar** [comment]
So the patch is working ?

**Comment by SwoopX** [comment]
DDF is working as expected, my Sonos SL is consuming 5W

```
"20:32:57:445": {
        "e": "changed",
        "id": "182",
        "r": "sensors",
        "state": {
            "lastupdated": "2022-10-02T18:32:57.435",
            "power": 5
        },
        "t": "event",
        "uniqueid": "54:ef:44:10:00:4d:da:0e-15-000c"
    }
```

![grafik](https://user-images.githubusercontent.com/4005212/193470565-f9b0c9e4-d78c-4a00-9618-344dfdaae0b6.png)

**Comment by grimmaldus** [comment]
> So the patch is working ?
My working solution was to alter the value right in the HA for the sensor. 

I wanted to do it via  ddf file within Home Assistant however there is no mapping for the addon container to this day to my knowledge (if you know any other way how to do it).  I will try to locate it right within container and alter it via DDF file and let you know.

**Comment by erik3rik** [comment]
I have the exact same case, originally reported as a bug of the deCONZ addon to Home Assistant, but since that is only wrapping the deCONZ software the fault is in the underlying deCONZ rather than the addon itself.

https://github.com/home-assistant/addons/issues/2679

Power values increased with a 10x after the update, and was "fixed" by rolling back (to deCONZ 2.17.1 from 2.18.2)

![Screenshot 2022-09-24 at 11 48 10](https://user-images.githubusercontent.com/46067475/194083536-ff2906e8-0f1b-443e-b0be-3d66bb78b4da.png)

**Comment by grimmaldus** [comment]
> @manup so we can close this issue?

I have not have time to test ddf since I'm away from home atm but I will do it this weekend.
